.ci/publish-to-gh-pages.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 
-# Download the latest release
-curl -s https://api.github.com/repos/jellyfin/jellyfin-docs/releases/latest | grep "browser_download_url.*docs-.*\.tar\.gz" | cut -d : -f 2,3 | tr -d \" | wget -O /tmp/docs.tar.gz -qi -
```

Comment:
you need to replace this with some thing like:

```sh
azure_build_id="$( az pipelines runs list --branch release-10.5.z --pipeline-ids 1 --reason batchedCI --result succeeded --status completed --query-order QueueTimeDesc --top 1 | jq '.[].id' )"
az pipelines runs artifact download --artifact-name 'Jellyfin Server Setup' --path ${installer_tmpdir} --run-id ${azure_build_id}
```

Reply:
Something like that or can I just use that exact command? It looks fine to me.

Reply:
It need different arguments, but yes it's the exact command that the build server uses to pull in the windows setup artifact.